features/OJP001_Relative-placement-for-elements-aggregated-to-another-element.feature

```diff
@@ -7,7 +7,7 @@ with an PlacementRelTo attribute pointing to the IfcLocalPlacement of the contai
 
   Scenario: Agreement on the relative placement of IfcElements being a part of another IfcElement through the relationship IfcRelAggregates
 
-      Given A file with Schema Identifier "IFC2X3"
+      Given A file with Schema Version "IFC2X3"
```

Comment:
Is there a reason to use `Version `here?

Reply:
No, I overlooked it; it's from an older commit. 
It should be just `'Schema'`
Changed it in 
https://github.com/buildingSMART/ifc-gherkin-rules/pull/114/commits/f479248f5606cb74e06fc2f0acdb86a7bb525512